config/default.yml

```
@@ -1491,7 +1491,7 @@ Lint/BinaryOperatorWithIdenticalOperands:
 Lint/BooleanSymbol:
   Description: 'Check for `:true` and `:false` symbols.'
   Enabled: true
-  Safe: false
+  SafeAutoCorrect: false
   VersionAdded: '0.50'
   VersionChanged: '0.83'
```

Can you update it and add a changelog entry?

```suggestion
  VersionChanged: '<<next>>'
```

---

Updated.